gdk-pixbuf2 cannot find -lstaticpixbufloader-gdiplus

I installed a few version of msys2 on 64bit Windows 10 (1803) to build a gtk3 application.

I had the gdk-pixbuf2 2.38.0-1 package installed.

When I compiled I received:

`gcc -o gtk3.exe gtk3.c $(pkg-config --cflags --libs gtk+-3.0)
/usr/lib/gcc/x86_64-pc-msys/7.3.0/../../../../x86_64-pc-msys/bin/ld: cannot find -lstaticpixbufloader-gdiplus`

I could not find any reference to a library called staticpixbufloader anyone on my pc.

I downgraded to gdk-pixbuf2 2.36.9-1 and was able to compile and run successfully.

I can see that https://gitlab.gnome.org/GNOME/gdk-pixbuf/blob/gdk-pixbuf-2-36/gdk-pixbuf/meson.build#L107 contains a reference to this library

--- a/mingw-w64-gdk-pixbuf2/PKGBUILD
+++ b/mingw-w64-gdk-pixbuf2/PKGBUILD
@@ -6,7 +6,7 @@ _realname=gdk-pixbuf2
 pkgbase=mingw-w64-${_realname}
 pkgname="${MINGW_PACKAGE_PREFIX}-${_realname}"
 pkgver=2.38.0
-pkgrel=1
+pkgrel=2
 pkgdesc="An image loading library (mingw-w64)"
 arch=('any')
 url="https://www.gtk.org/"
@@ -75,6 +75,8 @@ package() {
 
   sed -s "s|$(cygpath -m ${MINGW_PREFIX})|${MINGW_PREFIX}|g" -i "${pkgdir}${MINGW_PREFIX}"/lib/pkgconfig/gdk-pixbuf-2.0.pc
   sed -s "s|$(cygpath -m ${MINGW_PREFIX})|${MINGW_PREFIX}|g" -i "${pkgdir}${MINGW_PREFIX}"/share/thumbnailers/gdk-pixbuf-thumbnailer.thumbnailer
+  # https://github.com/Alexpux/MINGW-packages/issues/4726
+  sed -s "s| -lstaticpixbufloader-gdiplus||g" -i "${pkgdir}${MINGW_PREFIX}"/lib/pkgconfig/gdk-pixbuf-2.0.pc
 
   install -Dm644 "${srcdir}/gdk-pixbuf-${pkgver}/COPYING" "${pkgdir}${MINGW_PREFIX}/share/licenses/${_realname}/COPYING"
 }

